Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -311,7 +313,10 @@ - (void)ensureContentConstraints
         [rightButton autoPinEdgeToSuperviewEdge:ALEdgeBottom],
     ];
 
-    [self layoutIfNeeded];
+    // Layout immediately, unless the input toolbar hasn't even been laid out yet.
+    if (self.bounds.size.width > 0 && self.bounds.size.height > 0) {
```

Comment:
This condition avoids some misleading "iOS auto layout constraints cannot be satisfied" logging.